pydantic/fields.py

```diff
@@ -213,6 +221,8 @@ def __init__(self, **kwargs: Unpack[_FieldInfoInputs]) -> None:
         self.serialization_alias = kwargs.pop('serialization_alias', None)
         alias_is_set = any(alias is not None for alias in (self.alias, self.validation_alias, self.serialization_alias))
         self.alias_priority = kwargs.pop('alias_priority', None) or 2 if alias_is_set else None
+        self.field_title_generator = kwargs.pop('field_title_generator', None)
+        self.title_priority = kwargs.pop('title_priority', None) or 2 if self.title is not None else None
```

Comment:
Very minor nit - can we order these in the same order as seen in the attribute docstring?